src/nuget.config

```diff
@@ -8,7 +8,6 @@
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="particular myget.org" value="https://www.myget.org/F/particular/" />
-    <add key="Local Integration Source" value="..\nugets" />
```

Comment:
Why did you remove this? I'm not sure how the Integration solution will be able to build properly without this, but maybe I just don't understand how it's used?